api/conf/conf.yaml

```diff
@@ -22,6 +22,8 @@ conf:
   etcd:
     endpoints: # supports defining multiple etcd host addresses for an etcd cluster
       - 127.0.0.1:2379
+    username: ""
```

Comment:
if we use an empty string here, does it mean we could connect ETCD anonymously?

Reply:
that is not a good way

Reply:
Yes. When the string is empty, it can be linked anonymously